androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/AlertCard.kt

```diff
@@ -34,7 +34,8 @@ enum class AlertCardSpec {
     Major,
     Downstream,
     Secondary,
-    Elevator
+    Elevator,
+    Delay
```

Comment:
I was on the fence about adding `Delay` as the AlertCardSpec or `Minor`, but `Delay` seemed to make the logic the simplest.